src/svg-pan-zoom.js

```diff
@@ -436,12 +460,21 @@ SvgPanZoom.prototype.handleMouseDown = function(evt, prevEvt) {
     }
   }
 
+  this.isDown = true;
```

Comment:
There's `this.state === 'pan'` used across the codebase. Can we use that instead of creating one more param?